src/internal/components/dropdown/index.tsx

```diff
@@ -120,15 +119,19 @@ const TransitionContent = ({
       }
       onMouseDown={onMouseDown}
     >
-      <div className={clsx(styles['dropdown-content-wrapper'], isRefresh && styles.refresh)}>
-        <div className={styles['ie11-wrapper']}>
-          <div ref={verticalContainerRef} className={styles['dropdown-content']}>
-            <DropdownContextProvider position={position}>
-              {header}
-              {children}
-              {footer}
-            </DropdownContextProvider>
-          </div>
+      <div
+        className={clsx(
+          styles['dropdown-content-wrapper'],
+          !header && !children && styles['is-empty'],
```

Comment:
This is the line that I moved, the `is-empty` styles were being applied to the wrong node